backend/endpoints/rooms.py

```diff
@@ -132,11 +131,10 @@ async def remove_member(
             status_code=status.HTTP_404_NOT_FOUND,
             detail="admin id specified not a member of the room",
         )
-    
+
     if admin_id is not None and member_id == admin_id:
         raise HTTPException(
-            status_code=status.HTTP_403_FORBIDDEN,
-            detail="cannot remove yourself"
+            status_code=status.HTTP_403_FORBIDDEN, detail="cannot remove yourself"
```

Comment:
Based on slack behavior and the compiled zurichat document, there are no restrictions for members leaving a channel. A public channel can be completely empty.

The only restriction for leaving channels is for an empty private channel. And in that case the last member is prompted to archive the channel. The last member doesn't have to be the creator or admin.

I think this endpoint needs to be reworked to reflect that behavior.

Reply:
I'm not restricting anyone from leaving the channel, members can still leave the channel. Due to the nature of the endpoitn with is solving two problems leaving a channel now has 2 ways. I'm just making the consumer of the endpoint to just one way of implementing leaving a channel.